### PR TITLE
adding new apis under system.runtime.interopservices

### PR DESCRIPTION
**Pull request by mairaw**
@tijoytom, @luqunl who should review the docs for these new classes (OSPlatform and Architecture)?

Adding @joperezr and @AlexGhiondea for the RuntimeInformation class

### CODE REVIEW

File: xml/System.Runtime.InteropServices/RuntimeInformation.xml [hunk 1]
```diff
@@ -92,8 +106,8 @@
         <ReturnType>System.Runtime.InteropServices.Architecture</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the platform architecture on which the current app is running.</summary>
+        <value>The platform architecture on which the current app is running.</value>
```
Comment (joperezr) [inline xml/System.Runtime.InteropServices/RuntimeInformation.xml]: NIT: Should we add remarks here as we did above of the possible values? (Arm64, Arm, x64, x86)

File: xml/System.Runtime.InteropServices/RuntimeInformation.xml [hunk 2]
```diff
@@ -116,8 +130,8 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a string that indicates the name of the operating system on which the app is running.</summary>
+        <value>The name of the operating system on which the app is running.</value>
```
Comment (joperezr) [inline xml/System.Runtime.InteropServices/RuntimeInformation.xml]: NIT: I'm not sure `name` is the right word here, more like it is the description of the OS. Foro example, if running in .NET Native, we will return `Microsoft Windows` which not necessarily means that this is the name of the OS.

**Review by joperezr — APPROVED** [review]
Added two very small nits, but RuntimeInformation looks good otherwise.

**Review by luqunl — APPROVED** [review]
LGTM

**Review by BillWagner — APPROVED** [review]
This LGTM.

Once you've addressed @joperezr comments, :shipit:.

**Pull request merged**